GitUp/GitUp.xcodeproj/project.pbxproj

```diff
@@ -485,7 +486,7 @@
 		E21DCAEC1B253847006424E8 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				GCC_DYNAMIC_NO_PIC = YES;
+				CODE_SIGN_IDENTITY = "Mac Developer";
```

Comment:
Does this try and default to your current profile?

Reply:
Yes. The main app is set the same way, this is for the command-line tool.

Reply:
The annoyance here though is that the Tool target will complain when I build for the Application scheme. Is this a necessary change, seems more annoying that useful, unless I'm missing something.

Reply:
> The annoyance here though is that the Tool target will complain when I build for the Application scheme.

What warning are you seeing, I don't see one?

Reply:
FWIW signing Command-Line Tools is now required.

Reply:
EDIT: Ah, the `DEVELOPMENT_TEAM` needed to be moved to the project level. I run my branches with an additional commit changing the Development Team to mine — since I'm not a member of the team and I can't build at all otherwise — and then delete it before pushing, so I did not catch it. This will be completed in one last push. :)

Reply:
Done